ui/ducks/swaps/swaps.js

```diff
@@ -1212,7 +1211,7 @@ export const signAndSendTransactions = (
       hardware_wallet_type: getHardwareWalletType(state),
       stx_enabled: smartTransactionsEnabled,
       current_stx_enabled: currentSmartTransactionsEnabled,
-      stx_user_opt_in: smartTransactionsOptInStatus,
+      stx_user_opt_in: getSmartTransactionsOptInStatusForMetrics(state),
```

Comment:
Same question as above in terms of why repeat the selector instead of put this in a variable?

Reply:
Hi @martahj, this is a different method and we only call this selector once in the method.

Reply:
Sorry, I guess I misread the change